extensions/wikia/Email/Controller/DiscussionController.class.php

```diff
@@ -356,3 +356,47 @@ protected static function getEmailSpecificFormFields() {
         return array_merge_recursive( parent::getEmailSpecificFormFields(), $formFields );
     }
 }
+
+class DiscussionArticleCommentController extends DiscussionController {
+
+	private $contentType;
+	private $articleTitle;
+
+	const ARTICLE_COMMENT_REPLY = 'article-comment-reply';
+	const ARTICLE_COMMENT_AT_MENTION = 'article-comment-at-mention';
+	const ARTICLE_COMMENT_REPLY_AT_MENTION = 'article-comment-reply-at-mention';
+
+    public function initEmail() {
+		$this->articleTitle = $this->request->getVal( 'threadTitle' );
+		$this->contentType = $this->request->getVal( 'contentType' );
+
+		parent::initEmail();
+	}
+
+	protected function getSummary() {
+		return $this->getMessage(
+			$this->getTranslationKey(),
+			$this->getCurrentUserName(),
+			$this->articleTitle,
+			$this->wiki->city_title
+		);
+	}
+
+	protected function getSubject() {
+		return $this->getSummary();
+	}
+
+	private function getTranslationKey() {
+    	// TODO: Follower email
```

Comment:
What about this TODO?

Reply:
I wanted to make a task for this later, because this specific type of notification would require a lot more work.

Reply:
You know, I will describe it better. It really looks like a comment that should be resolved during normal development.